apps/course-authoring/scripts/pipeline/setup.mjs

```diff
@@ -0,0 +1,50 @@
+import fs from '../utils/file-system.mjs';
```

Comment:
We're not using this file anywhere, right? I couldn't find any reference of the file of functions.

Also, we are using the `@liascript/simple-scorm-packager` instead of the `scorm-again`, is it going to break anything if we keep the `node_modules/scorm-again/dist/scorm12.js` in the `fileMap`?

Reply:
Added comments explaining what this file does. I would like to leave it in as an example incase we need this type of functionality in the future.